prepare_assets.sh

```diff
@@ -129,6 +129,13 @@ elif [[ "${OS_NAME}" == "windows" ]]; then
 else
   cd vscode || { echo "'vscode' dir not found"; exit 1; }
 
+  if [[ "$VSCODE_ARCH" == "ppc64le" ]]; then
```

Comment:
Can you move that branch into `check_tags.sh` after `# linux-armhf`?
I think the `.tar.gz` should still be generated, no?

Reply:
I've moved it to `check_tags.sh`, but shouldn't we also export this on `build/build.sh`?

Reply:
No, since it becomes env variables (in the workflow) with `echo "SHOULD_BUILD_REH=${SHOULD_BUILD_REH}" >> "${GITHUB_ENV}"`